### PR TITLE
[Related #33] Create panel component

### PR DESCRIPTION
**Pull request by supunTE**
Related #33 Panel Component

### CODE REVIEW

File: apps/client/src/components/organisms/panel/Panel.component.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      onClick={(): void => setCollapse(preValue => !preValue)}>
+      <div
+        className={cs(
+          { 'gap-20 overflow-y-hidden': isCollapse },
+          { 'overflow-y-scroll scrollbar': !isCollapse },
+          'flex flex-col pr-5',
+          'transition-all duration-300',
+          'gap-5',
+        )}
+        id='scrollPanel'>
```
Comment (dalanad) [inline apps/client/src/components/organisms/panel/Panel.component.tsx:45]: Why do we need a `id` here

File: apps/client/tailwind.config.js [hunk 2]
```diff
@@ -13,6 +13,10 @@ module.exports = {
       text: {
         light: '#F4DDDB',
       },
+      bg:{
+        brown: '#3B2D2C',
+        'brown-100': '#6B5A59',
```
Comment (dalanad) [inline apps/client/tailwind.config.js:18]: Do we have other brown variants in the design? If so shall we include them all

File: apps/client/src/components/organisms/panel/Panel.component.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
+import { ReactElement, useEffect, useState } from 'react';
+import { Description, Text } from '../..';
+import { ReactComponent as ArrowHeadTop } from 'pixelarticons/svg/chevron-up.svg';
```
Comment (dalanad) [inline apps/client/src/components/organisms/panel/Panel.component.tsx]: Rename to `ArrowHeadTopIcon`

**Review by dalanad — CHANGES_REQUESTED** [review]

File: apps/client/src/components/organisms/panel/Panel.component.tsx (hunk 1 above)
Comment (supunTE) [reply apps/client/src/components/organisms/panel/Panel.component.tsx:45]: The id is used to scroll the panel to the top when collapsing.

 ```
   useEffect(() => {
    const element = document.getElementById('scrollPanel');
    const handleScroll = (): void => {
      element?.scrollTo({ top: 0, behavior: 'smooth' });
    };
    handleScroll();
    return () => {
      window.removeEventListener('scroll', handleScroll);
    };
  }, [isCollapse]);
  ```

File: apps/client/src/components/organisms/panel/Panel.component.tsx (hunk 3 above)
Comment (supunTE) [reply apps/client/src/components/organisms/panel/Panel.component.tsx]: fixed https://github.com/the-ai-team/razor/pull/41/commits/6609129235ab29039445a1013b4a2c9e3e9da0e8

File: apps/client/tailwind.config.js (hunk 2 above)
Comment (supunTE) [reply apps/client/tailwind.config.js:18]: These were changed in PR #43 under commit https://github.com/the-ai-team/razor/pull/43/commits/c50411361f07279c7eda2da51c2712b1b41c052a

**Review by dalanad — APPROVED** [review]

**Review by pasinducw — APPROVED** [review]

**Pull request merged**